test/model/src/core/util.cpp

```diff
@@ -283,6 +284,61 @@ shared_memory::~shared_memory()
     }
 }
 
+/*
+ * convert_charset --
+ *     Perform a character set conversion. Throw an exception on error.
+ */
+std::string
+decode_charset_bytes(const std::string &str, const char *src_charset)
+{
```

Comment:
There are only a small set of a single byte character sets, and afaict this has really been written to support UTF-8. Perhaps it would be prudent to assert or throw if the character set is *not* UTF-8.

Reply:
This makes sense. Since we only need this for UTF-8, I just specialized the function for it and renamed it to `decode_utf8`. Fixed in [11095d9](https://github.com/wiredtiger/wiredtiger/pull/10551/commits/11095d9d11fb5293a1b0198346ca77db568cbf32). Thank you!